bin/prettier.js

```diff
@@ -171,17 +171,15 @@ if (stdin) {
 } else {
   eachFilename(filepatterns, filename => {
     fs.readFile(filename, "utf8", (err, input) => {
-      if (write) {
-        console.log(filename);
-      }
-
       if (err) {
```

Comment:
I think it makes sense, yes. It reinforces the file that is currently being worked on, and the user it most likely reading the list of files, not the error, at first.